.gitignore

```diff
@@ -1,5 +1,6 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
-/docs/build/
-/docs/Manifest.toml
+docs/build
+docs/src
```

Comment:
Huh?

Reply:
Okay, you are generating from weave into it? That is very non-standard and I don't think it should be in here.

Reply:
That's fine and fair. I think it is a cool practice that enhances Documenter.jl, but is fine if we keep it without it.